pgxgeom_test.go

```diff
@@ -161,8 +161,9 @@ func TestCodecScanValuePolymorphic(t *testing.T) {
 
 				err = conn.QueryRow(ctx, query, pgx.QueryResultFormats{format}).Scan(&point)
 				assert.Error(t, err)
-				err = err.(pgx.ScanArgError).Err
-				assert.Equal(t, err.Error(), "pgxgeom: got *geom.Polygon, want *geom.Point")
+				var scanArgError pgx.ScanArgError
+				assert.True(t, errors.As(err, &scanArgError))
+				assert.Equal(t, scanArgError.Err.Error(), "pgxgeom: got *geom.Polygon, want *geom.Point")
```

Comment:
Hmm, okay. I was intending to test only the part of the error message that is controlled by this package rather than the "can't scan into dest[0]" prefix that pgtype itself adds. But I've switched to `EqualError` with the full message; done, and let me know if you see a better way to do this.